### ISSUE TITLE
Remove disabled checkbox examples from Storybook

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#2630]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

We directly copied the examples from USWDS on checkbox and radio button which depict a disabled form element (the last element in the list, labeled George Washington Carver). We don't disable form elements at VA. These examples confuse designers who think that we have changed that policy. We have not changed that policy.

## What I expected to happen

All form field examples should be enabled. Our components should not allow for a disabled state.

## Reproducing

Steps to reproduce:

1. Look here and see a disabled option in a number of variations: https://design.va.gov/storybook/?path=/docs/uswds-va-radio--default

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by jamigibbs** [comment]
Closing as a duplicate. This appears to have been done here:

- https://github.com/department-of-veterans-affairs/component-library/pull/1330
